### PR TITLE
Add data about packaging

### PR DESCRIPTION
**Pull request by brentzundel**
this PR adds a common schema for Instrument of Trade, which is a term of art for packaging material.

### CODE REVIEW

**Review by mkhraisha — APPROVED** [review]

File: docs/openapi/components/schemas/common/InstrumentOfTrade.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          NPPO to the producer of the wood packaging material or treatment
+          provider who applies the marks or the entity otherwise responsible
+          to the NPPO for ensuring that appropriately treated wood is used and
+          properly marked (shown in the examples as “000”). The number and order
+          of digits and/or letters are assigned by the NPPO.
+        type: string
+      treatmentCode:
+        title: Treatment Code
+        description: >-
+          a treatment code using the appropriate abbreviation according to ISPM
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/InstrumentOfTrade.yml:50]:
```suggestion
          A treatment code using the appropriate abbreviation according to ISPM
```

**Review by TallTed — APPROVED** [review]

**Comment by nissimsan** [comment]
@brentzundel , "InstrumentOfTrade" seems like a re-invention of https://vocabulary.uncefact.org/LogisticsTransportEquipment . We already have this on Trace: 
https://w3c-ccg.github.io/traceability-vocab/#TransportEquipment . I recommend using it instead, extending as needed.

**Comment by brentzundel** [comment]
@nissimsan Transport Equipment seemed to be shipping containers and trucks, rather than pallets and boxes. Are they really the same thing? If so I can extend transport equipment to include packing material and wood treatment stamps, but they seemed at different layers to me.

**Comment by nissimsan** [comment]
@brentzundel , 

You commonly distinguish returnable vs non-returnable. 

Transport Equipment is returnable. This includes pallets: 
https://vocabulary.uncefact.org/containedTransportEquipment
"A piece of transport equipment contained within this piece of logistics transport equipment, such as a pallet."

If you are looking for non-returnable, consider: 
https://vocabulary.uncefact.org/Package

It doesn't seem like it should be necessary to invent a term here.

**Review by mprorock — APPROVED** [review]

**Comment by TallTed** [comment]
@nissimsan — It seems worth noting that pallets are often *not* returnable (or at least, not *returned*), leading to endless ads for "[FREE PALLETS](https://www.google.com/search?q=free+pallets)!"

It seems that a loaded-and-wrapped pallet in a "[shipping container](https://www.google.com/search?q=shipping+container)" that gets loaded from dock to ship to railcar to semi-trailer *could* be interpreted as *either* a [Package](https://vocabulary.uncefact.org/Package) _or_ a piece of [containedTransportEquipment](https://vocabulary.uncefact.org/containedTransportEquipment), by the linked definitions. 

(I find it odd that they have [contained Transport Equipment](https://vocabulary.uncefact.org/containedTransportEquipment), but no [***un***contained Transport Equipment](https://vocabulary.uncefact.org/uncontainedTransportEquipment).)

All of which to say... UNCEFACT's ontology seems to be less than fully thought through, or at least, less than clearly documented. Confusion seems to be built in.

**Review by nissimsan — APPROVED** [review]

**Pull request merged**